torch/onnx/symbolic_helper.py

```diff
@@ -0,0 +1,284 @@
+import torch
+from torch._C import ListType
+import warnings
+
+import torch.onnx
+# This import monkey-patches graph manipulation methods on Graph, used for the
+# ONNX symbolics
+import torch.onnx.utils
+
+from functools import wraps
+
+
+# EDITING THIS FILE? READ THIS FIRST!
+#
+# - This file is ONLY for ATen operators (e.g., operators that show up in the
```

Comment:
I left this block here to avoid duplicating it in all symbolic_opset<version>.py files.
Should I slightly modify it here and reference it in the other files or would it be preferable to copy it in both symbolic_opset9.py and symbolic_opset10.py?

Reply:
You can keep most part here, and leave the reference in the symbolic_opset9.py and symbolic_opset10.py